README.md

```diff
@@ -1,26 +1,36 @@
-Bitcoin Core integration/staging tree
+Namecoin Core integration/staging tree
 =====================================
 
-[![Build Status](https://travis-ci.org/bitcoin/bitcoin.svg?branch=master)](https://travis-ci.org/bitcoin/bitcoin)
+https://namecoin.info/
```

Comment:
Agreed with @josephbisch .  Also I don't see a reason to remove the Travis CI build status.  Our status is at the same URL, just with namecoin/namecore rather than bitcoin/bitcoin .